.travis.yml

```diff
@@ -48,12 +47,10 @@ before_install:
   wget https://repo.continuum.io/miniconda/Miniconda$PY_MAJOR-latest-$OS-x86_64.sh -O miniconda.sh
 - bash miniconda.sh -b -p $HOME/miniconda;
 - export PATH="$HOME/miniconda/bin:$HOME/miniconda/lib:$PATH";
-- hash -r;
 - conda config --set always_yes yes --set changeps1 no;
-- conda create -y -q -n test_cq -c cadquery -c conda-forge pythonocc-core=0.18.2 oce=0.18.2 python=$TRAVIS_PYTHON_VERSION
-- source ~/miniconda/bin/activate test_cq;
-- pip install .[dev]
-- python -c 'import OCC.gp as gp; print(gp.gp_Vec())'
+- conda env create -f environment.yml
+- source ~/miniconda/bin/activate cadquery
+- conda install -c conda-forge -c defaults -c cadquery python=$PYTHON_VERSION
```

Comment:
This line should have `cadquery=2` in it instead of just `cadquery` shouldn't it? That's what's in the readme for installation, correct?

Reply:
`-c cadquery` means use the cadquery channel. BTW this is CI so one shouldn't install cadquery from a conda package.